src/Properties/launchsettings.json

```diff
@@ -1,10 +1,12 @@
 {
   "profiles": {
     "DevHome (Package)": {
-      "commandName": "MsixPackage"
+      "commandName": "MsixPackage",
+      "commandLineArgs": "--utilitylaunch DevHome.PI.Exe",
+      "nativeDebugging": true

```

Comment:
don't check this line in, but maybe leave the nativeDebugging line.

Reply:
Good catch. I'll remove both of them, as native debugging is a slower than pure managed (more symbols load)